### PR TITLE
Add documentation about tree shaking

### LINKED ISSUE #267: Add docs about babel-plugin-date-fns
See: https://github.com/date-fns/babel-plugin-date-fns and https://github.com/date-fns/date-fns/issues/179#issuecomment-261828789

### PR DESCRIPTION
**Pull request by xkizer**
Fixes #267

### CODE REVIEW

File: docs/gettingStarted.md [hunk 1]
```diff
@@ -61,6 +61,28 @@ import addDays from 'date-fns/fp/utc/addDays'
...
+
+Then add to your `.babelrc`:
+```
+{
+  "plugins": ["date-fns"],
+  "presets": ["es2015"]
+}
+```
+
+You can not write the following and the plugin will do the rest:
```
Comment (kyhy) [inline docs/gettingStarted.md]:
```suggestion
You can write the following and the plugin will do the rest:
```

**Comment by TrySound** [comment]
It's not treeshaking. It converts all named imports to path imports to fix the lack of treeshakability in the code or in the bundle. Treeshaking is code analysing with eliminating unused stuff, not hack like suggested one.

File: docs/gettingStarted.md [hunk 2]
```diff
@@ -61,6 +61,28 @@ import addDays from 'date-fns/fp/utc/addDays'
 import { addDays, format } from 'date-fns/fp'
 ```
 
+## Tree shaking with babel
+If you use date-fns with babel, you can improve treeshakability without having to import from multiple files. This
+can be achieved by using the [date-fns babel plugin](https://github.com/date-fns/babel-plugin-date-fns).
+
+First, install the date-fns babel plugin:
```
Comment (dcousens) [inline docs/gettingStarted.md]: I don't think you need these instructions.  They are verbatim from https://github.com/date-fns/babel-plugin-date-fns and if users are interested,  they can visit that repository for the instructions.

**Comment by leshakoss** [comment]
I would love to see date-fns+babel documentation, but probably not in gettingStarted.md. I am closing the PR because it is old enough that I don't expect any more activity here but feel free to reopen it if you wish.

**Pull request closed without merging**